app/main.py

```diff
@@ -1,13 +1,10 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {"line": error["line_number"], "column": error["column_number"], "message": error["text"], "name": error["code"], "source": "flake8"}
```

Comment:
For all functions: beautify your return, make it look like:
return {
               key: value,
               key: value,
               key: value
           }